public/static/app/perf-bb/SignalProcessingCtrl.js

```diff
@@ -24,23 +24,17 @@ mciModule.controller('SignalProcessingCtrl', function(
   // Holds currently selected items
   vm.selection = [];
 
-  // Could not be overridden by persistent user settings
-  const mandatoryDefaultFiltering = {
+  vm.defaultChangePointFilter = {
```

Comment:
Moved these onto the controller so they could be tested.

Reply:
it's probably too much work for this PR but [OutlierState](https://github.com/evergreen-ci/evergreen/blob/d5f2b29b2891857476df9942b14ba60020843d49/public/static/app/perf-bb/PerfBBOutliersCtrl.js#L371) was refactored out for testing purpose in the PerfBBOutliersCtrl.js

Reply:
I would say that is outside the scope of this work.

Reply:
For sure, it's not worth the return. But it might be worth raising a ticket for it or stating that it is in this one.